presto-benchmark-runner/src/main/java/com/facebook/presto/benchmark/framework/BenchmarkRunnerConfig.java

```diff
@@ -85,4 +87,16 @@ public BenchmarkRunnerConfig setJsonEventLogFile(String jsonEventLogFile)
         this.jsonEventLogFile = Optional.ofNullable(jsonEventLogFile);
         return this;
     }
+
+    public boolean getProceedWithFailures()
```

Comment:
nit: since it returns a boolean, maybe `isProccedWithFailures`? 

Or maybe this config could be names `proceedWithFailuresEnabled`? I don't have a strong opinion on config name change.

Reply:
Leiqing suggested we use `continueOnFailure` and `isContinueOnFailure`. I made changes accordingly.